documentation/high-avail/ha-api-hld.md

```diff
@@ -105,6 +108,7 @@ HA set is defined as a SAI object and contains the following SAI attributes:
 | SAI_HA_SET_ATTR_DP_CHANNEL_PROBE_INTERVAL_MS | `sai_uint32_t` | The interval of the data plane channel probe. |
 | SAI_HA_SET_ATTR_DP_CHANNEL_PROBE_FAIL_THRESHOLD | `sai_uint32_t` | The threshold of the data plane channel probe fail. |
 | SAI_HA_SET_ATTR_DP_CHANNEL_IS_ALIVE | `bool` | (Read-only) Is data plane channel alive. |
+| SAI_HA_SET_ATTR_SWITCHOVER_NETWORK_CONVERGENCE_TIME_MS | sai_uint32_t | Time for which DPU driven HA state machine needs to wait for the network to switchover traffic during planned shutdown of the other DPU in the HA pair. |
```

Comment:
This field looks to be missing in the HA detailed design, we will need to add it there as well.

also:

- nit: "_IN_MS".
- might be better to rephrase this part: "Time for which DPU driven HA state machine to wait ...". Slightly hard to read it.
- Since it is switchover, we don't need to mention planned shutdown here, so better remove this part.
- Since this one will be DPU driven mode only, so will be better to update the attribute to say DPU_DRIVEN_HA_SWITCHOVER_NETWORK_CONVERGENCE_TIME_IN_MS (Or I prefer simply DPU_DRIVEN_HA_SWITCHOVER_WAIT_TIME_IN_MS)

Reply:
done